crates/core_arch/src/x86/bt.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(test)]
+use stdsimd_test::assert_instr;
+
+/// Returns the bit in position `b` of the memory addressed by `p`.
+#[inline]
+#[cfg_attr(test, assert_instr(bt))]
+#[unstable(feature = "simd_x86_bittest", issue = "0")]
```

Comment:
Sorry that it took so long to review this again. Is there a PR for the `simd_x86_bittest` feature to rust-lang/rust ? Otherwise, I'd prefer to use the `stdsimd` feature here for now I think. Or what do you think @alexcrichton ? If we wanted to stabilize this in the near future, then adding their own feature flag might make more sense.

Reply:
Eh I think it'd be best to move away from `stdsimd` where we can, so I think landing this as a different feature name is fine. Before landing though we'll want to open an issue at rust-lang/rust to track this and fill in the issue number though

Reply:
I've opened a tracking issue https://github.com/rust-lang/rust/issues/59414 

@mehcode can you update the issue number to 59414 ?